### PR TITLE
Add white background to logo

### PR DESCRIPTION
**Pull request by purva-thakre**
## Description
Related to the work introduced in #416.

This PR makes sure the `toqito` logo is visible in dark mode of docs. 

Inverted color version of the logo does not look that good. Could be tackled later in another PR as we want to be consistent with the color theme of `wagtail`. Added these images to try a customized version of CSS where the logo image is switched depending on which theme is being used: light/dark.

https://www.reddit.com/r/tailwindcss/comments/kd9czo/comment/gfw0zxb/?utm_source=share&utm_medium=web2x&context=3

Or we don't need to tackle this. Use the image with a white background.

## Todos
Notable points that this PR has either accomplished or will accomplish.
  -  [ ] TODO 1

## Questions
-  [ ] Question1

## Status
-  [ ] Ready to go

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# ![logo](./docs/figures/logo.svg "logo")
+# ![logo](./docs/figures/logo.png "logo")
```
Comment (purva-thakre) [inline README.md:1]: Update this link to fix broken image in PyPI.

https://stackoverflow.com/a/46875147/10241324

image
https://pypi.org/project/toqito/

**Comment by purva-thakre** [comment]
marking this ready for review even though I have 1 item I need to resolve. 

I can't use a permalink to fix that comment because the new images I want to use are in this PR.

File: README.md (hunk 1 above)
Comment (purva-thakre) [inline README.md:1]:
```suggestion
# ![logo](https://github.com/vprusso/toqito/raw/cfb62c4a5ce04b782f64229e7acd2b1c97f09801/docs/figures/logo.svg)
```

Comment (purva-thakre) [reply README.md]: Temporarily use existing svg image. Will fix this later.

**Comment by Bchass** [comment]
Did the inverted colors not work?

**Comment by purva-thakre** [comment]
yeah, they don't really look as clean as the current logo. 

You can take a look at this commit: c663960eb5e6b5c3fc81d3faf531eb77c163728f

**Comment by purva-thakre** [comment]
@vprusso Can I go ahead and merge this? I think merging this PR would be helpful for the WIP changes in #416.

I think I figured out why the image on the `PyPI` page is broken. This PR handles it and hopefully we will see the logo after a later release.

image

We can worry about customizing the logo for the new theme later.

**Comment by vprusso** [comment]
> @vprusso Can I go ahead and merge this? I think merging this PR would be helpful for the WIP changes in #416.
> 
> I think I figured out why the image on the `PyPI` page is broken. This PR handles it and hopefully we will see the logo after a later release.
> 
> image
> 
> We can worry about customizing the logo for the new theme later.

Yep, merge away!

**Pull request merged**